cmd/cli/kubectl-kyverno/utils/filter/filter.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 	"strings"
 
+	wildcard "github.com/IGLOU-EU/go-wildcard"
```

Comment:
```suggestion
	wildcard "github.com/kyverno/kyverno/pkg/utils/wildcard"
```

Please use our wrapper